docs/chianti.rst

```diff
@@ -44,6 +49,37 @@ should show the paths set in the configuration file.
 .. _SSW: http://www.lmsal.com/solarsoft/
 .. _HDF5: https://en.wikipedia.org/wiki/Hierarchical_Data_Format
 
+Testing Against IDL Routines
+----------------------------
+
+The `fiasco` test suite includes a set of tests that automatically compare against the equivalent routines in the
+CHIANTI IDL software.
+The purpose of these tests is to provide a systematic way to assess any deviations from the original IDL software.
+To run these tests,
+
+.. code-block::shell
+
+   $ pytest --idl-executable=/path/to/idl \
+            --idl-codebase-root=/path/to/chianti/idl \
+            --ascii-dbase-root=/path/to/chianti/dbase \
+            --include-all-files \
+            fiasco/tests/idl/
+
+where ``/path/to/idl/`` is the path to the directory containing ``bin/idl`` (where ``idl`` is the IDL executable),
+``/path/to/chianti/idl`` is the path to the directory containing all of the CHIANTI IDL routines,
+and ``/path/to/chianti/dbase`` is the path to the top directory of the CHIANTI atomic database (what you would usually
```

Comment:
```suggestion
where :file:`/path/to/idl/` is the path to the directory containing :file:`bin/idl` (where ``idl`` is the IDL executable),
:file:`/path/to/chianti/idl` is the path to the directory containing all of the CHIANTI IDL routines,
and :file:`/path/to/chianti/dbase` is the path to the top directory of the CHIANTI atomic database (what you would usually
```
[optional] The `:file:` role will let you have a different style for file paths than for literals, which we ended up doing for PlasmaPy in a CSS file during a round of procrastination-driven development. It's not really important, though.